thirdeye/thirdeye-frontend/app/pods/manage/explore/route.js

```diff
@@ -12,9 +12,14 @@ import { formatYamlFilter, redundantParse } from 'thirdeye-frontend/utils/yaml-t
 import moment from 'moment';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 
+const ANALYSIS_RANGE = [
+  moment().add(1, 'day').subtract(30, 'day').startOf('day').valueOf(),
+  moment().add(1, 'day').startOf('day').valueOf()
+];
+
```

Comment:
I know it's beyond the scope of this PR, but I think this sort of constant is a good candidate for being in the `thirdeye-frontend/tests/utils/constants` file:  ANALYSIS_30_DAY_RANGE
(with some descriptive explanation) 
... so it can be reused

Also, is 30 days really what's desired?  think Feb (28-day month) or 31-day months
would `.subtract(1, 'months')` make more sense?

Reply:
I think yes you are right. I think we may want to have it programatic based on the month. But I am not 100% sure yet, will need to confirm and see if there was a rationale to hardcode it to 30 in the first place. As you said, we don't want to address tangential things in this PR, so in one of the future PR's we'd address it.

Reply:
I think yes you are right. I think we may want to have it programatic based on the month. But I am not 100% sure yet, will need to confirm and see if there was a rationale to hardcode it to 30 in the first place. As you said, we don't want to address tangential things in this PR, so in one of the future PR's we'd address it.

Reply:
OK.  I agree with postponing the `30-day` hardcoding.  But, I will would like to have it moved to the constants file.  
I am seeing that it is use in 2 other places and it would make sense you just `re-`use it. In addition, if we decide to change the hardcoded aspect, then it will be done in one place and applied everywhere it is used

Reply:
I would not want to commingle any refactoring effort into this RB. It is confusing to reviewers. This should be part of another PR.